recipes/nss/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "3.36.1" %}
+{% set version_us = version.replace(".", "_") %}
+
+package:
+  name: nss
+  version: {{ version }}
+
+source:
+  fn: nss-{{ version }}.tgz
+  url: https://ftp.mozilla.org/pub/security/nss/releases/NSS_{{ version_us }}_RTM/src/nss-{{ version }}.tar.gz
+  sha256: 6025441d528ff6a7f1a4b673b6ee7d3540731ada3f78d5acd5c3b3736b222bff
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - autoconf
+    - cmake
+    - nspr
+    - sqlite
+    - zlib
+    - libgcc
```

Comment:
We shouldn't need `libgcc` here as we are just using the system copy of the compiler run time currently.